docs/2fa/users/guide.md

```diff
@@ -0,0 +1,100 @@
+# User Guide
+
+<!-- Visit [Automata 2FA Guru](https://app.2fa.guru) to start your 2FA protection. -->
+Visit Automata 2FA Guru to start your 2FA protection.
+
+---
+
+## Setup 2FA
+<!-- - Visit [Automata 2FA Guru](https://app.2fa.guru). -->
+- Visit Automata 2FA Guru.
+
+- Use metamask and connect your wallet.
+
+- If your account hasn't signed up, you can click the **Wallet Actions** button on the top, and go through the signup process. You need to sign a message to prove your identity before generating a 2FA secret. Attention, you must use a different device to store the 2FA secret, don't keep the 2FA device and your wallet in the same one.
+
+![](../../assets/2fa/setup-1.png)
+
+![](../../assets/2fa/setup-2.png)
+
+![](../../assets/2fa/setup-3.png)
+
+And we also support 1password for the 2FA secret representation. You need to prepare a **Login** item in 1password, configure the basic information, then you can use **Scan QR Code** function in 1password. It will make it more flexiable and efficient.
```

Comment:
We support 1Password for storing your 2FA secret. To enable the use of **Scan QR Code** function in 1Password, you would first have to prepare a **Login** item in 1Password and configure the basic information. This makes password retrieval more convenient.